tensorrt_llm/_torch/pyexecutor/py_executor.py

```
@@ -2036,6 +2082,78 @@ def _broadcast_sample_state_loop(self):
+                        release_with_tag(*tags)
+                        torch.cuda.synchronize()
+                        gc.collect()
+                        torch.cuda.empty_cache()
+                    elif action == "wakeup":
+                        materialize_with_tag(*tags)
+                        torch.cuda.synchronize()
+                    else:
+                        error_msg = f"unknown action '{action}'"
+                        logger.warning(
+                            f"Control listener: {error_msg}, ignoring.")
+                except Exception as exc:
```

Ruff (0.15.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)